src/components/chapter/ChapterList.tsx

```diff
@@ -191,6 +164,10 @@ export default function ChapterList(props: IProps) {
             </Fab>
         ));
 
+    if (noChaptersFound) {
+        makeToast('No chapters found', 'warning');
+    }
+
```

Comment:
This will blast the toast on every render

Reply:
noChaptersFound is false by default so it would just check for the boolean on every render